components/topbar/user/test/index.test.js

```diff
@@ -32,6 +32,19 @@ describe('topbar/user', () => {
     expect(getByText(/custom content/i)).to.exist
     expect(queryByTitle(navCTA.text)).not.exist
   })
+
+  it('The user can open toggable menus', async () => {
+    const {queryByText} = setup()
+
+    const firstNavButtonElement = queryByText(navMain[0].label)
+
+    fireEvent.click(firstNavButtonElement)
+
+    await waitFor(() => {
+      const firstLinkItemElement = queryByText(navMain[0].menu[0].links[0].text)
+      expect(firstLinkItemElement).to.have.attribute('href', '#link-1')
+    })
+  })
 })
 
 function setup(customProps) {
```

Comment:
🍆  Could/should we have constant data in an external file? (fixtures or something similar)

Reply:
Done!